autorag/evaluation/metric/generation.py

```diff
@@ -330,11 +330,11 @@ def sem_score(
 	generations = [metric_input.generated_texts for metric_input in metric_inputs]
 	generation_gt = [metric_input.generation_gt for metric_input in metric_inputs]
 	if embedding_model is None:
-		embedding_model = embedding_models["huggingface_all_mpnet_base_v2"]()
+		embedding_model = embedding_models["huggingface_all_mpnet_base_v2"]
```

Comment:
Need fix here. This is fixed at the other issue. 
Maybe you couldn't pull the main branch after merge (Why no conflict here? strange)
I can fix it. (Don't worry)